lms/static/sass/_build-lms.scss

```diff
@@ -63,6 +63,7 @@
 @import "discussion/elements/editor";
 @import "discussion/elements/labels";
 @import "discussion/elements/navigation";
+@import "discussion/views/home";
```

Comment:
This new partial file removes a section of the deprecated _discussion.scss file (shown in the next file below) to help clarify that these are still actively used styles. The new partial accounts for existing lines 256-416 in _discussion.scss